### PR TITLE
Allow header disclaimer and logo to be skinned

### PR DESCRIPTION
**Pull request by jonathaningram**
If you're interested, see our matching PR that matches this change: https://github.com/govau/ci-console/pull/3

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ const config = {
     modules: ['node_modules'],
     // Required for some module configs which use these fields
     // See https://github.com/flatiron/director/issues/349
-    mainFields: ['browserify', 'browser', 'module', 'main']
+    mainFields: ['browserify', 'browser', 'module', 'main'],
+
+    symlinks: false
```
Comment (jcscottiii) [inline webpack.config.js]: just curious why was this needed?

**Comment by jcscottiii** [comment]
@jonathaningram LGTM. just a question

File: webpack.config.js (hunk 1 above)
Comment (jonathaningram) [reply webpack.config.js]: This allows us to symlink our skin into the `skins` directory:

```
$ ls -ls static_src/skins/
total 8
0 drwxr-xr-x  3 me  staff  102 13 Oct 09:09 cg
8 lrwxr-xr-x  1 me  staff   56  5 Oct 14:52 govau -> /Users/me/go/src/github.com/govau/ci-console/skin/
```

And we are symlinking (at the very least) for development: our skin is in another repo and I symlink it up.

Without this change, the UI won't build, giving the error:

```
ERROR in /Users/me/go/src/github.com/govau/ci-console/skin/index.js
Module build failed: ReferenceError: Unknown plugin "transform-object-rest-spread" specified in "base" at 0, attempted to resolve relative to "/Users/me/go/src/github.com/govau/ci-console/skin"
<omitted>
```

Upgrading babel and friends could fix this, but this config change does fix it and doesn't appear to affect anything else.

**Comment by mogul** [comment]
What's the question? :)

File: webpack.config.js (hunk 1 above)
Comment (jcscottiii) [reply webpack.config.js]: thanks for that! @jonathaningram could you make an issue to revisit this when babel is upgraded?

**Comment by jonathaningram** [comment]
@jcscottiii rebased. Also I updated circle.yml (see https://github.com/18F/cg-dashboard/pull/1260/commits/4822c34401924ef98a6939caa71a111ca8d29893) to install latest chromedriver. I needed this to fix my failing integration tests for this PR (which worked locally).

This hopefully also fixes flakiness as you can see in the failed master build: https://circleci.com/gh/18F/cg-dashboard/2654
